lib/transformers/allOf.js

```diff
@@ -45,13 +44,72 @@ function generateErrorSummary(transformationOrder, transformations) {
         : '';
 }
 
-function allOf({schema, options, transform, data, schemaErrors} = {}) {
-    const opts = merge({}, options);
-    const groupSchema = schema.allOf[0];
-    const pageHeading = groupSchema.title;
+function schemaContent(schema, nunjucksImports, summaryText, transformations) {
+    const componentsSchema = schema.allOf;
+    function propertyNunjucks(propertySchema) {
+        const {properties} = propertySchema;
+        const propertyIds = Object.keys(properties);
+        return propertyIds
+            .map(propertyId => {
+                const transformation = transformations[propertyId];
+                if (
+                    transformation.componentName === 'rawContent' ||
+                    transformation.componentName === 'summary'
+                ) {
+                    return transformation.content;
+                }
+                return nunjucksHelper.toNunjucks(
+                    transformation.componentName,
+                    transformation.macroOptions
+                );
+            })
+            .join('\n');
+    }
+
+    function beginFieldset(componentSchema) {
```

Comment:
could these functions all be defined at the top-level scope here? we want do avoid having functions defined in functions where not needed.
`beginFieldset` and `componentContent` functions should be at the top level

Reply:
Moved to top level scope